Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -62,6 +69,14 @@ section FixedPoints
 
 variable {M α}
 
+@[to_additive] theorem mem_fixedPoints_iff_subsingleton_orbit {a : α} :
+    a ∈ fixedPoints M α ↔ (orbit M a).Subsingleton := by
+  rw [mem_fixedPoints]
+  constructor
+  · rintro h _ ⟨m, rfl⟩ y ⟨p, rfl⟩
+    simp only [h]
+  · exact fun h m ↦ h (mem_orbit a m) (mem_orbit_self a)
```

Comment:
This is a new lemma, right? Can it move to a more appropriate PR?

Reply:
removed